hail/python/setup-hailtop.py

```diff
@@ -17,8 +17,9 @@
     package_dir={
         'hailtop': 'hailtop'},
     package_data={
-        "hailtop": ["py.typed"],
-        'hailtop.hailctl': ['hail_version', 'deploy.yaml']},
+        "hailtop": ["py.typed", "hail_version"],
+        'hailtop.hailctl': ['hail_version', 'deploy.yaml']
```

Comment:
Why does `hailctl` need `hail_version` if it's getting it through `hailtop`? I'm not super well-versed in the wild west of python package management.

Reply:
I'm not too sure, this code is already in main: 
https://github.com/hail-is/hail/blob/022d02f2d4322f26103d1242454279ad398e6032/hail/python/setup-hailtop.py#L21

I just formatted the bracket and added extra lines per Dan's suggestion: https://github.com/hail-is/hail/pull/10085#issuecomment-793064791

Reply:
Ok we'll leave it in for now then